project/.travis/before_install_test.sh.twig

```diff
@@ -5,9 +5,16 @@ if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
     mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp
     echo "memory_limit=3072M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
+    {% if 'mongodb' in services %}
     if [ ${TRAVIS_PHP_VERSION} '<' '7.0' ]; then
         echo "extension=mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    else
+        echo "extension=mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+
+        # Backwards compatibility with old mongo extension
+        composer require "alcaeus/mongo-php-adapter" --no-update
```

Comment:
Why requiring this package here? Should not be added directly on the `composer.json` project file?